ui_template/ui.html

```diff
@@ -67,6 +67,30 @@
                     </div>
                 </div>
 
+                <div id="mb-3">
+                    <label class="form-label"
+                        >ユーザー辞書のエクスポート&インポート</label
+                    >
+                    <div class="form-text">辞書のエクスポートをします。</div>
+                    <a
+                        class="btn btn-primary m-3"
+                        href="/download_dict"
+                        target="_blank"
+                        rel="noopener noreferrer"
+                    >
```

Comment:
[GET]`/user_dict`でもユーザー辞書のダウンロードができるので、そちらを利用するようにするとエンドポイントを増やす必要が無いので良いかと思います。

```suggestion
                    <a
                        class="btn btn-primary m-3"
                        href="/user_dict"
                        target="_blank"
                        rel="noopener noreferrer"
                        download="user_dict.json"
                    >
```